src/Esprima/JavascriptParser.cs

```diff
@@ -98,24 +118,17 @@ public Context()
     /// <summary>
     /// Creates a new <see cref="JavaScriptParser" /> instance.
     /// </summary>
-    /// <param name="code">The JavaScript code to parse.</param>
-    public JavaScriptParser(string code) : this(code, new ParserOptions())
+    public JavaScriptParser() : this(ParserOptions.Default)
     {
     }
 
     /// <summary>
     /// Creates a new <see cref="JavaScriptParser" /> instance.
     /// </summary>
-    /// <param name="code">The JavaScript code to parse.</param>
     /// <param name="options">The parser options.</param>
     /// <returns></returns>
-    public JavaScriptParser(string code, ParserOptions options)
+    public JavaScriptParser(ParserOptions options)
```

Comment:
That could work too but I like to keep parameterless ctors if possible because they could come in handy in some situations (e.g. `new()` generic constraint exists only for parameterless ctors).